src/sass/components/_intro-story.scss

```diff
@@ -125,17 +118,6 @@
 }
 
 
-.background--image1 {
-  @include breakpoints (small) {
-    background-image: url('/img/global/intro-story/slide1-flagpole-mountains-698x600.jpg');
```

Comment:
Can this image (and some of the others) be removed from /img/global/intro-story/?